internal/pkg/service/common/etcdop/iterator/iterator_test.go

```diff
@@ -108,7 +109,7 @@ ETCD_REQUEST[%d] ✔️️  GET ["some/prefix/", "some/prefix0") | rev: %d | cou
 			expectedLogs: `
 ETCD_REQUEST[%d] ➡️  GET ["some/prefix/", "some/prefix0")
 ETCD_REQUEST[%d] ✔️️  GET ["some/prefix/", "some/prefix0") | rev: %d | count: 4 | %s
-ETCD_REQUEST[%d] ➡️  GET ["some/prefix/foo004", "some/prefix0")
+ETCD_REQUEST[%d] ➡️  GET ["some/prefix/foo004", "some/prefix0") | rev: %d
 ETCD_REQUEST[%d] ✔️️  GET ["some/prefix/foo004", "some/prefix0") | rev: %d | count: 1 | %s
```

Comment:
By default, iterator nacitava vsetky 2+ stranky, s rovnakou reviziou ako mala prva stranka.
To sa nelogovalo, teraz sa to loguje, tak preto tieto zmeny.